scout/supporting_data/config_schema.yml

```diff
@@ -139,7 +139,7 @@ properties:
 
       alt_ref_carb:
         type: ["string", "null"]
-        enum: [MidCaseTCExpire, DECARB-bau, null]
+        enum: [MidCase, DECARB-bau, null]
         description: Specify alternate reference grid scnenario for baseline electricity emissions intensities (rather than AEO). 'MidCaseTCExpire' refers to the Standard Scenarios Mid Case and 'DECARB-bau' refers to the ReEDS "DECARB" business-as-usual scenario.
```

Comment:
Description needs to be updated for `MidCase` instead of `MidCaseTCExpire`

Reply:
Yes, will do in subsequent commit.